cmd/podman/registry/registry.go

```diff
@@ -99,7 +99,7 @@ func GetContext() context.Context {
 
 func DefaultAPIAddress() string {
 	if rootless.IsRootless() {
-		xdg, err := util.GetRuntimeDir()
+		xdg, err := util.GetRootlessRuntimeDir()
```

Comment:
Depending on how consolidation of `IsRootless` goes, this might eventually be able to call the c/storage version directly.